aksk/pkg/iam/driver/keystone/keystone.go

```diff
@@ -220,3 +244,58 @@ func (iam *KeystoneIam) DownloadAkSk(ctx context.Context, in *pb.GetAkSkRequest)
 	json.Unmarshal(bodyBytes, &akskListout)
 	return akskListout, nil
 }
+
+func isValidUserId(userId string, token string, host string) bool {
+
+	// Validate userId is legitimate
+	client := &Keystone.Client
+	keystoneURL := PROTOCOL + host + USER_DETAILS
+	getreq, err := http.NewRequest(GET, keystoneURL+userId, bytes.NewBuffer(nil))
+	if err != nil {
+		log.Error("Error in validating the UserId")
+		return false
+	}
+	getreq.Header.Add(AUTH_TOKEN, token)
+	getreq.Header.Set(CONTENT_TYPE, APPL_JSON)
+
+	var validationResponse *http.Response
+	validationResponse, err = client.Do(getreq)
+	if err != nil {
+		log.Error("Error in validating TenantId", err)
+		return false
+	}
+
+	if validationResponse.StatusCode == 200 {
+		log.Info("UserId is Valid")
+		return true
+	}
+
+	return false
+}
+
+func isValidTenantId(tenantId string, token string, host string) bool {
```

Comment:
IMO, Comment for ```isValidUserId``` method also apply here

Reply:
fixed

Reply:
isValidTenantId or isValidUserId is more or less same. Only one method is enough. You may remove any of these.